src/js/edu-benefits/5490/config/form.js

```diff
@@ -124,9 +165,9 @@ const formConfig = {
             }
           }
         },
-        previousBenefits: {
-          title: 'Previous Benefits',
-          path: 'benefits-eligibility/previous-benefits',
+        benefitHistory: {
+          title: 'Benefit history',
```

Comment:
Same trouble with the missing title here.

![image](https://cloud.githubusercontent.com/assets/12970166/24121478/9e0147d0-0d75-11e7-8e30-54ba87d95ef9.png)